src/pydvl/influence/general.py

```diff
@@ -2,32 +2,15 @@
 Contains parallelized influence calculation functions for general models.
```

Comment:
As opposed to functions, here a subject _is_ needed 😅 

Also, this module should be called "common" for consistency with the rest of the library